### PR TITLE
Improve user documentation

### PR DESCRIPTION
**Pull request by brainwane**
Followup to #382.

### CODE REVIEW

File: docs/userhelp/source/account-help.rst [hunk 1]
```diff
@@ -9,18 +9,21 @@ This depends on whether you are a healthcare provider or not.
...
-Go to the main page and click the "Register New Account" link.
+Go to the main page and click the "Register New Account" link. The site
+will help you create an account that can submit enrollment applications
+for individual or organizational healthcare providers.
 
-A new account needs a username, first and last name, a user role, and an
-email address. The site will create a random password and email it to
-you, and will assume that the account is for a provider.
+A new account needs a username, first and last name, a role (your title
+within your organization), and an email address. The site will create a
```
Comment (jasonaowen) [inline docs/userhelp/source/account-help.rst]: I don't understand "role" in this context. If you click on the "Register New Account", you get this screen:

screenshot 2017-09-12 16 05 18

What does "the title within your organization" refer to?

**Comment by brainwane** [comment]
This is now ready for review.

File: docs/userhelp/source/account-help.rst (hunk 1 above)
Comment (brainwane) [reply docs/userhelp/source/account-help.rst]: That role reference is a remnant of documentation meant for system admins. Removed.

File: docs/userhelp/source/enrollment.rst [hunk 2]
```diff
@@ -1,18 +1,15 @@
 Enrollment
 ==========
 
-This help section explains what an enrollment is, tells you what you
-need to do to create one and submit it, and answers common questions
-about confusing terms.
+What information do I need to apply to enroll?
```
Comment (cecilia-donnelly) [inline docs/userhelp/source/enrollment.rst]: Nice rewording.

**Comment by chj124** [comment]
i have a comment on the use of terms in the PSM documentation
clip from DESIGN.md about the Service agent

Service agents should be able to create, view, and edit enrollments for their provider clients.

Can view: provider dashboard, enrollments, profile
Can create an enrollment
Can edit draft enrollments
Can view submitted enrollments

should be:

Can view: provider dashboard, applications, enrollments, profile
Can create an application
Can edit draft applications
Can view submitted applications
---
A service agent does not create or edit enrollments.
A service agent can enter, update, or view applications on a providers behalf
Part of the application process is to select the plans/services the provider wants to participate in and is able to do

enrollment is what the state staff would be responsible for. the validate the info from the app and approve the eligibility. No one else besides the Service Admin should have access to approve or edit enrollment information.

the provider and agent can view the data only.

**Comment by cecilia-donnelly** [comment]
Thanks @chj124!  I'm linking to this comment from issue #10, where we're having a related discussion.

**Comment by brainwane** [comment]
@chj124 Thank you for your clarification!

You mentioned:

> A service agent does not create or edit enrollments.
> A service agent can enter, update, or view applications on a provider's behalf

I would appreciate it if you could help me understand what the difference is between **creating an enrollment** and **entering an application**. Is the difference that a provider or a service agent can create an **application**, but only a service admin can create an **enrollment**?

Thank you.

**Comment by chj124** [comment]
so, a provider who is practicing in the state (or possible on a bordering town in another state) needs to complete an application. This is a request to be considered to provide medicaid service to the states clients.

This is only a application, and until they are vetted and approved, they may not submit a bill for delivering services under our medicaid programs.

Once the person responsible reviews the application and verifies the data/credentials, then the application status is approved. At this point the person responsible for plan enrollments would have to review the providers information and verify there is a need for the provider. Just because they apply doesn't mean they are enrolled. There are some limits on the number of providers allowed for some provider types.

At that point, if the need is there, the information is correct, and the provider has signed the appropriate agreement, then the enrollment data is created with the start date (when claim submission is allowed) and possibly and end date (date by which update info needs to be gathered and all data reviewed to ensure they still qualify)

does that help?

**Pull request merged**